buzzmobile/sense/controller/src/controller_node.cpp

```diff
@@ -10,35 +10,34 @@
 
 using namespace buzzmobile;
 
+// Buttons mapped to a PS4 controller using the Joy ROS node.
 #define reverse_button buttons[0]
-#define brake_button buttons[14]
+#define brake_button buttons[2]
 #define horn_button buttons[1]
 #define velocity_trigger axes[4]
 #define toggle_auto_button buttons[12]
 
 ros::Publisher motion_pub;
 ros::Publisher state_pub;
-//NOTE: this is also published by the autonomous brake...this is ok.
-ros::Publisher brake_pub;
+
+const float MAX_TURNING_ANGLE = M_PI / 6; // 30 degrees in radians
 
 int maxFwdSpeed = 1; //m/s
 int pubFreq     = 10; //hz
 
 bool obstacleFlag = false;
 
+// Keep track of the values to continously publish
```

Comment:
Hm. Idk what happens if we go with either case. If we don't continuously publish a message to the motor, won't the motor get the message and then stop doing what it was doing because it didn't get a message right after telling it to continue at it?

If this doesn't happen, then I agree this thing isn't needed.